Nos/Controller/NoteEditorController.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 import SwiftUI
 import UIKit
 
-/// A controller for Nostr note text that is being edited. This controllers pairs with a `NoteUITextViewRepresentable`
+/// A controller for Nostr note text that is being edited. This controller pairs with a `NoteUITextViewRepresentable`
 /// to help our SwiftUI views interact with a UITextView cleanly.
 /// 
 /// To use: instantiate and pass into a `NoteTextEditor` view. You can retrieve the typed text via the `text` property
```

Comment:
This one worked for me before just the way it was, and now that you've showed me, I want these magic links everywhere!

```suggestion
/// To use: instantiate and pass into a ``NoteTextEditor`` view. You can retrieve the typed text via the `text` property
```

Reply:
I like these too! But since they're not strictly typos, what if I did a separate PR and did tons of them throughout the app?

Reply:
Sure, works for me!